rust/cubesql/cubeclient/src/models/v1_load_request_query.rs

```diff
@@ -7,20 +9,21 @@ pub struct V1LoadRequestQuery {
     #[serde(rename = "segments", skip_serializing_if = "Option::is_none")]
     pub segments: Option<Vec<String>>,
     #[serde(rename = "timeDimensions", skip_serializing_if = "Option::is_none")]
-    pub time_dimensions: Option<Vec<crate::models::V1LoadRequestQueryTimeDimension>>,
+    pub time_dimensions: Option<Vec<super::V1LoadRequestQueryTimeDimension>>,
     #[serde(rename = "order", skip_serializing_if = "Option::is_none")]
-    pub order: Option<Vec<Vec<String>>>,
+    pub order: Option<Value>,
```

Comment:
This is the main change here, to temporarily support both variants of `order`.
All other changes are to have a better idiomatic Rust code!